Added three new commands: StandEvery, StandEnable, StandDisable. (#2)
mvllow/stand.nvim: readme.md

`StandWhen` - Display time remaining until next stand

`StandNow` - Stand, dismissing any notifications and restarting the timer

> `StandEvery` - Set the break timer interval in minutes. Will apply to next timer

Reviewer: Nitpick to phrase this in a similar tone as the other commands.

```suggestion
`StandEvery` - Set the stand interval in minutes for future timers
```

Author: Totally reasonable :)
